README.md

```diff
@@ -638,12 +647,15 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 * [**Magisk**](https://github.com/topjohnwu/Magisk)
 * [**TWRP**](https://twrp.me)
+* [**OrangeFox**](https://orangefox.download)
+* [**PitchBlackRecovery**](https://pitchblackrecovery.com)
 
 ------------------------
 
 ## – Tutorials and Guides –
 
 * [**OTA on Rooted Device**](tutorials/Install-Rooted-OTA.md)
+* [**Reverse Engineering**](https://github.com/OshekharO/Reverse-Engineering/wiki)
```

Comment:
I'm not sure if this really fits a Open Source list.

Reply:
Yup it does fit in  tutorials and guide